api/api.yaml

```diff
@@ -0,0 +1,249 @@
+
+  version: "1.0.0"
+  title: "Tafel Missions API"
+  x-audience: "company-internal"
+  x-api-id: "AD1CB381-1612-422B-A322-19D4F1DFFE45"
+  contact:
+    name: "Github Issues"
+    url: "https://github.com/prambrucke/tafel/issues"
+host: "TBD"
+schemes:
+- "https"
+
+paths:
+  /teams/{team-id}/missions:
+    get:
+      description: |
+        Retrieves missions for the givesn team-id
+      # security:
+      # - oauth2:
+      #   - tafel-missions.read
+      produces:
+      - "application/json"
+      - "application/problem+json"
+      parameters:
+        - $ref: '#/parameters/team-id'
+      responses:
+        200:
+          description: "Successful operation"
+          schema:
+            $ref: '#/definitions/MissionList'
+        default:
+          description: Error occurred - see status code and problem object for information
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /teams/{team-id}/missions/{mission-id}:
+    get:
+      description: |
+        Retrieves  mission details for the given tema-id and mission-id.
+      # security:
+      # - oauth2:
+      #   - balance-service.read
+      produces:
+      - "application/json"
+      - "application/problem+json"
+      parameters:
+        - $ref: '#/parameters/team-id'
+        - name: mission-id
+          in: path
+          required: true
+          type: string
+      responses:
+        200:
+          description: "Successful operation"
+          schema:
+            $ref: '#/definitions/Mission'
+        default:
+          description: Error occurred - see status code and problem object for information
+          schema:
+            $ref: '#/definitions/Problem'
+  
+    post:
+     description: |
+      Creates a new issue
+     produces:
+      - "application/json"
+      - "application/problem+json"
+     parameters:
+        - $ref: '#/parameters/team-id'
+        - name: mission-id
+          in: path
+          required: true
+          type: string
+        - name: "mission"
+          in: body
+          description: "Mission request"
+          schema:
+            $ref: '#/definitions/Mission'
+     responses:
+        201:
+          description: "Successful operation"
+          schema:
+            $ref: '#/definitions/Mission'
+        default:
+          description: Error occurred - see status code and problem object for information
+          schema:
+            $ref: '#/definitions/Problem'
+ 
+    put:
+     description: |
+      Updates a mission
+     produces:
+      - "application/json"
+      - "application/problem+json"
+     parameters:
+        - $ref: '#/parameters/team-id'
+        - name: mission-id
+          in: path
+          required: true
+          type: string
+        - name: "mission"
+          in: body
+          description: "Mission request"
+          schema:
+            $ref: '#/definitions/Mission'
+     responses:
+        200:
+          description: "Successful operation"
+          schema:
```

Comment:
PUT need not return anything, i feel!

Reply:
I think returning 204, would result in another network call to get the updated content.